src/main/java/com/questhelper/steps/QuestStep.java

```diff
@@ -506,4 +513,91 @@ public BufferedImage getQuestImage()
 	{
 		return spriteManager.getSprite(SpriteID.TAB_QUESTS, 0);
 	}
+
+
+	public void showWorldTooltips(PanelComponent panelComponent, boolean showHovered)
```

Comment:
nit/naming: rename `showHovered` variable to `isMenuOpen`, since that's what makes the decision - and we might as well make that clear in this function
nit/naming: `showWorldTooltips` doesn't describe well what this actually does, maybe something along the lines of `renderBackgroundQuestSourceTooltip` or something, maybe a bit long though